Microsoft.Toolkit.Win32/Microsoft.Windows.Interop.WindowsXamlHost.WinForms/Interop/Win32/SetWindowPosFlags.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace MS.Win32
```

Comment:
Should be `Microsoft.Toolkit.Win32.UI.Interop.WinForms.Interop.Win32`

Reply:
@nmetulev This name seems a little strange. Since it contains Win32 interop for winforms, should we consider something like `Microsoft.Toolkit.WinForms.UI.Interop` with Win32/WinRT then as suffix?